code/datums/supplypacks.dm

```diff
@@ -1646,6 +1646,26 @@ var/list/all_supply_groups = list(supply_emergency,supply_security,supply_engine
 					)
 	containername = "shield control board crate"
 
+///////////// Bathroom Fixtures
+
+/datum/supply_packs/misc/toilet
+	name = "Urination Station Crate"
```

Comment:
No, but I couldn't think of a better crate name since "Relief crate" (the original name I came up with) didn't feel like it was obvious what it contained. Open to suggestions.